WORKING_GROUPS.md

```diff
@@ -248,3 +249,18 @@ Responsibilities include:
   to be generated when needed.
 * Defining and adding common structures to the dumps generated
   in order to support tools that want to introspect those dumps.
+
+### [Release](https://github.com/nodejs/release)
+
+The Release Working Group manages the release process for Node.js.
+Responsibilities include:
+
+Responsibilities include:
```

Comment:
Remove one of the "Responsibilities include:"  :-D

Reply:
HA! Sigh

Reply:
Commit messages is :100: on this one.